src/lockers/OCY/OCY_Convex_A.sol

```diff
@@ -260,10 +260,9 @@ contract OCY_Convex_A is ZivoeLocker, ReentrancyGuard {
             uint256 extraRewardsLength = IBaseRewardPool_OCY_Convex_A(convexRewards).extraRewardsLength();
             for (uint256 i = 0; i < extraRewardsLength; i++) {
                 address rewardContract = IBaseRewardPool_OCY_Convex_A(convexRewards).extraRewards(i);
-                uint256 rewardAmount = IBaseRewardPool_OCY_Convex_A(rewardContract).rewardToken().balanceOf(
-                    address(this)
-                );
-                if (rewardAmount > 0) { IERC20(rewardContract).safeTransfer(OCT_YDL, rewardAmount); }
+                IERC20 rewardToken = IBaseRewardPool_OCY_Convex_A(rewardContract).rewardToken();
```

Comment:
Should be 
`IBaseRewardPool_OCY_Convex_A(rewardContract).rewardToken().token();`

as for OCY_Convex_C.sol below

Reply:
Please ignore the above comment.
The fix is indeed correct, since the convex pool is of id < 151